clang/include/clang/Basic/Attr.td

```diff
@@ -974,7 +974,8 @@ def AnalyzerNoReturn : InheritableAttr {
   // vendor namespace, or should it use a vendor namespace specific to the
   // analyzer?
   let Spellings = [GNU<"analyzer_noreturn">];
-  // TODO: Add subject list.
+  let Args = [DefaultBoolArgument<"Value", /*default=*/1, /*fake=*/0>];
```

Comment:
~What's the purpose of the `analyzer_noreturn` argument?~
EDIT: nvm, I learned it in the `HACK` comment of `isImmediateSinkBlock`.